scripts/run-integration-tests.sh

```diff
@@ -8,7 +8,16 @@ pushd "$(dirname "${BASH_SOURCE[0]}")" > /dev/null
 [[ ${UTIL_SOURCED-no} != yes && -f ./util.sh ]] && source ./util.sh
 popd > /dev/null
 
-ENV_NAME=$@
+PARAMETERS=$@
+PARAMETER_ARRAY=($PARAMETERS)
+ENV_NAME=${PARAMETER_ARRAY[0]}
+APPSTREAM_ENABLED=${PARAMETER_ARRAY[1]}
```

Comment:
Should parameter key and value also reflect egress?

Reply:
updated